padertorch/contrib/examples/source_separation/tasnet/README.md

```diff
@@ -12,19 +14,26 @@ With the default parameters, the following numbers can be obtained:
 
 | trained with  | SI-SDRi  | SDRi  |
 |---|---|---|
-| default  | 16.4  | 16.7  |
-| `with log_mse`  | 16.7  | 17.0  |
+| DPRNN default  | 16.4  | 16.7  |
+| DPRNN `with log_mse`  | 16.7  | 17.0  |
+
```

Comment:
we should add at least some results for the STFT/ISTFT Encoder/Decoder, since they should achieve only slightly worse results

Reply:
Yes. I planned to run this experiment after the test one with the TasNet encoder and decoder finishes. Or do you already have numbers produces with this model and implementation? If so, you can add them.

Reply:
I already have numbers, but not with your evaluation. To allow for a fair comparison it should be trained and evaluated with this code. 
I would add one or two sentences describing the STFT/ISTFT Enocer/Decoder and how to use them to the Readme.

Reply:
I can add those now if you don't mind